docs/diagnostics/DiagnosticSuppression.md

```diff
@@ -0,0 +1,39 @@
+# Diagnostic Suppression
+If a diagnostic is not relevant to a specific code fragment and you have an explanation of why it is not relevant, you can suppress the diagnostic so that it is not displayed for the code fragment.
+
+You can suppress Acuminator diagnostics in one of the following ways (which are described in greater detail below):
+
+ - By adding a specific comment directly in code
+ - By adding a code fragment to the Acuminator suppression file
+
+## Diagnostic Suppression by Adding a Code Comment
+You can suppress any diagnostic by adding a specific comment directly in code. To add a suppression comment, you select **Suppress the PX10YY diagnostic with Acuminator > in a comment** (where PX00YY is the code of the diagnostic) as a code fix for the diagnostic, as shown in the following screenshot.
```

Comment:
The PX10YY is not correct - it can be any Acuminator  diagnostic ID. And the second reference to it is "PX00YY"
```suggestion
You can suppress any diagnostic by adding a specific comment directly in code. To add a suppression comment, you select **Suppress the PX**** diagnostic with Acuminator > in a comment** (where PX**** is the code of the diagnostic) as a code fix for the diagnostic, as shown in the following screenshot.
```

Reply:
Which diagnostic has PX00YY code? We have no such diagnostics publicly described (see summary.md)

Reply:
Changed to PXYYYY